P2553/P2553.md

```diff
@@ -18,6 +18,34 @@ toc: true
 
 # Revision History
 
+## R2
+
+- address feedback from LWG small group review on 2022/05/31
+    - generally throughout, add context to annotations like "change paragraph 8"
+    - rename `SizeT` to `SizeType` and `OtherSizeT` to `OtherSizeType`,
+        to avoid confusion with `size_t`
+    - mandate that each element of `extents`'s parameter pack `Extents`
+        is either equal to `dynamic_extent`,
+        or is a representable value of type `SizeType`
+    - don't specify that the exposition-only @_are-valid-extents_@ are public
+    - for all the exposition-only @_are-valid-extents_@,
+        say "is nonnegative" instead of using code
+    - remove redundant condition on second @_are-valid-extents_@
+    - in [mdspan.extents.ctor], don't use @_are-valid-extents_@;
+        just inline the wording, because each @_are-valid-extents_@ overload
+        is (was) only used once anyway
+    - remove exposition-only functions @_are-valid-extents_@ entirely,
+        since they are no longer used anywhere (please see above changes)
+    - paragraph 1 change to [mdspan.extents.comp] could be separated from P2553,
+        so move it to P0009
+    - use correct stable name [mdspan.layoutleft.ctor]
+    - use correct stable name [mdspan.layoutright.ctor]
+    - use correct stable name [mdspan.layoutstride.ctor],
+        and fix spelling and paragraph number
+    - Move wording change to [mdspan.layoutstride.ctor]
```

Comment:
I'll delete this comment, since the relevant wording for this constructor now just refers to the constructor above it.